recipes/torchvision-extra-decoders/recipe.yaml

```
@@ -0,0 +1,79 @@
+context:
+  name: torchvision-extra-decoders
+  version: "0.0.2"
+  # https://github.com/pytorch-labs/torchvision-extra-decoders/issues/22
+  gitsha: b6d9e5c8fae88f9980a9faaf0b8c71787ca5b193
+
+package:
+  name: ${{ name|lower }}
```

```suggestion
  name: ${{ name }}
```

`lower` doesn't seem to do anything here?

---

I'm just goign to remove this indirection. i really dislike the additional mental toll it brings.